projects/holo/services/holo/tests/holo.nix

```
@@ -0,0 +1,60 @@
+{
+  sources,
+  ...
+}:
+
+{
+  name = "holo";
+
+  nodes = {
+    machine =
+      { ... }:
+      {
+        imports = [
+          sources.modules.ngipkgs
+          sources.modules.programs.holo
+          sources.modules.services.holo-daemon
+          sources.examples.holo.holo
+        ];
```

Inspired by the Ethersync test ;)

```suggestion
        ];
        services.getty.autologinUser = "root";
```

This will solve the auth issue.